rudder-jetty/SOURCES/patches/jetty/jetty-init-softokn-version-check.patch

```diff
@@ -1,17 +1,20 @@
-diff -ruN jetty7/bin/jetty-rpm.sh jetty7.new/bin/jetty-rpm.sh
---- jetty7/bin/jetty-rpm.sh	2017-04-25 11:22:09.988181290 +0200
-+++ jetty7.new/bin/jetty-rpm.sh	2017-04-25 11:28:08.012174878 +0200
-@@ -483,6 +483,14 @@
+--- jetty7/bin/jetty-rpm.sh.orig	2017-05-17 13:56:36.919856595 +0200
++++ jetty7/bin/jetty-rpm.sh	2017-05-17 14:01:52.135850950 +0200
+@@ -483,6 +483,18 @@
      UMASK="0007"
      echo "Setting umask to ${UMASK}"
  
 +    # Check if there is a mismatch in nss version, see http://www.rudder-project.org/redmine/issues/9905
 +    nss_ver=$(rpm -qa nss | cut -d - -f 2)
++    nss_bug_ver=$(echo "${nss_ver}" | cut -d . -f 1-2)
 +    softokn_ver=$(rpm -qa nss-softokn | cut -d - -f 3)
-+    if [ "${nss_ver}" != "${softokn_ver}" ]
++    if [ "${nss_bug_ver}" = "3.15" ] && [ "${nss_ver}" != "${softokn_ver}" ]
 +    then
 +      echo "nss version mismatch, your nss-softokn version (${softokn_ver}) should match your nss version (${nss_ver})"
 +      exit 1
++    elif [ "${nss_ver}" != "${softokn_ver}" ]
++    then
++      echo "nss version mismatch, your nss-softokn version (${softokn_ver}) should match your nss version (${nss_ver})" >> /var/log/rudder/webapp/java.log
```

Comment:
Maybe `your nss-softokn version (${softokn_ver}) may not work properly with your (${nss_ver})` or something like this.